docs/section10/10-1.md

```diff
@@ -28,16 +28,13 @@
 **Applicability Check 1:** 
 - Get dictionary list of proposed zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_p = get_hvac_zone_list_w_area (P_RMR)`
 - Get dictionary for finding baseline system types for hvac systems: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
-Get all systems in P_RMR with humidification specified in heating system or preheat system and get corresponding list of zones served by those systems.
+Get all systems in P_RMR with humidification specified and get corresponding list of zones served by those systems.
 - For each HVAC system in the P_RMR: `For hvac_p in P_RMR...HeatingVentilatingAirConditioningSystem:`  
-    - Reset hvac_has_humidication_p boolean variable to FALSE: `hvac_has_humidication_p = FALSE`
-    - For each heating system in hvac_p: `For heating_system_p in hvac_p.heating_system:`
-        - Check if the the humidification_type does not equal Null or None, if it does not equal Null or None then set the hvac_has_humidication_p boolean variable to True: `if heating_system_p.humidification_type != Null and heating_system_p.humidification_type != "None": hvac_has_humidication_p = TRUE` 
-    - For each preheating system in hvac_p: `For preheat_system_p in hvac_p.preheat_system:`
-        - Check if the the humidification_type does not equal Null or None, if it does not equal Null or None then set the hvac_has_humidication_p boolean variable to True: `if preheat_system_p.humidification_type != Null and preheat_system_p.humidification_type != "None": hvac_has_humidication_p = TRUE`  
-    - Check if hvac_has_humidication_p equals True, if it does then add the zones served to list of zones with humidification in the proposed: `if hvac_has_humidication_p == TRUE:`
+    - Reset hvac_has_humidification_p boolean variable to FALSE: `hvac_has_humidication_p = FALSE`  
+    - Check if the humidification_type does not equal Null or None, if it does not equal Null or None then set the hvac_has_humidification_p boolean variable to True: `if hvac_p.humidification_type != Null and hvac_p.humidification_type != "None": hvac_has_humidication_p = TRUE`  
+    - Check if hvac_has_humidification_p equals True, if it does then add the zones served to list of zones with humidification in the proposed: `if hvac_has_humidication_p == TRUE:`
```

Comment:
This line could just be: 

`if hvac_p.humidification_type != Null and hvac_p.humidification_type != "None":

& get rid of hvac_has_humidication_p